README.md

```diff
@@ -73,6 +76,38 @@ On Android permission is required to write in the external storage when saving t
 <application>
 ```
 
+Then you have to explicitly ask for the permission
```

Comment:
This code only works with ejected apps, and not with Expo React Native apps. I suggest also pointing users via a link to both [react-native's PermissionsAndroid](https://reactnative.dev/docs/permissionsandroid) and [Expo's Permissions](https://docs.expo.io/versions/latest/sdk/permissions/) APIs in addition to including sample code.

Reply:
Thanks for the links.

Do you mean I should include Expo's permission code snippet as well?

Reply:
I hope the changes reflect what is being expected. Let me know if you want if something is out of place.

Reply:
Good catch on the `let`. This looks good!